test_run_aca_review_function fails in ska3/flight as unpriv user

Running as jeanconn on fido

```
In [1]: import sparkles

In [2]: sparkles.test()
=================================================== test session starts ===================================================
platform linux -- Python 3.6.2, pytest-3.2.1, py-1.4.34, pluggy-0.4.0
rootdir: /proj/sot/ska3/flight/lib/python3.6/site-packages, inifile:
plugins: remotedata-0.3.0, openfiles-0.3.0, doctestplus-0.1.3, arraydiff-0.2
collected 22 items                                                                                                         

sparkles/tests/test_checks.py ...........
sparkles/tests/test_review.py ......F....

======================================================== FAILURES =========================================================
______________________________________________ test_run_aca_review_function _______________________________________________

    def test_run_aca_review_function():
        aca = get_aca_catalog(**KWARGS_48464)
        acar = aca.get_review_table()
        acars = [acar]
    
>       exc = run_aca_review(load_name='test', acars=acars)

sparkles/tests/test_review.py:180: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
sparkles/core.py:124: in run_aca_review
    roll_level=roll_level, loud=loud, obsids=obsids, context=context)
sparkles/core.py:158: in _run_aca_review
    report_dir.mkdir(parents=True, exist_ok=True)
../pathlib.py:1226: in mkdir
    self._accessor.mkdir(self, mode)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

pathobj = PosixPath('test_sparkles'), args = (511,)

    @functools.wraps(strfunc)
    def wrapped(pathobj, *args):
>       return strfunc(str(pathobj), *args)
E       PermissionError: [Errno 13] Permission denied: 'test_sparkles'

../pathlib.py:387: PermissionError
==================================================== warnings summary =====================================================
None
  could not create cache path /proj/sot/ska3/flight/lib/python3.6/site-packages/.cache/v/cache/lastfailed

-- Docs: http://doc.pytest.org/en/latest/warnings.html
==================================== 1 failed, 21 passed, 1 warnings in 77.45 seconds =====================================
Out[2]: 1
```

--- a/sparkles/tests/test_review.py
+++ b/sparkles/tests/test_review.py
@@ -172,12 +172,12 @@ def test_catch_exception_from_method():
         acar.run_aca_review(roll_level='BAD VALUE')
 
 
-def test_run_aca_review_function():
+def test_run_aca_review_function(tmpdir):
     aca = get_aca_catalog(**KWARGS_48464)
     acar = aca.get_review_table()
     acars = [acar]
 
-    exc = run_aca_review(load_name='test', acars=acars)
+    exc = run_aca_review(load_name=tmpdir.join('test'), acars=acars)
 
     assert exc is None
     assert acar.messages == [

